src/test/scala/chiseltest/iotesters/GCDTest.scala

```diff
@@ -44,7 +45,14 @@ object GcdRegression {
     val testValues = (for {x <- 2 to 100; y <- 2 to 100} yield (x, y, computeGcd(x, y)))
 
     val backendName = "verilator" // if (args.nonEmpty) { "verilator" } else { "treadle" }
+    val options = backendName match {
+      case "verilator" => Seq(VerilatorBackendAnnotation)
+      case "treadle" => Seq(TreadleBackendAnnotation)
+      case other => throw new RuntimeException(s"Unknown backend: $other")
+    }
 
+    // TODO
+    /*
     Driver.execute(Array(
       "--target-dir", "test_run_dir/iotesters_gcd",
       "--top-name", "iotesters_gcd",
```

Comment:
Seems like a pretty big TODO 🤔

Reply:
haha. I don't consider this (the ability to run more than one PeekPokeTester) a primary feature.